llvm/include/llvm/Bitcode/LLVMBitCodes.h

```diff
@@ -332,8 +332,6 @@ enum MetadataCodes {
   METADATA_INDEX = 39,                  // [bitpos]
   METADATA_LABEL = 40,                  // [distinct, scope, name, file, line]
   METADATA_STRING_TYPE = 41,      // [distinct, name, size, align, ...]
-  METADATA_FORTRAN_ARRAY_TYPE = 42, // [distinct, name, [bounds ...], ...]
-  METADATA_FORTRAN_SUBRANGE = 43, // [distinct, lbound, lbnde, ubound, ubnde]
   METADATA_COMMON_BLOCK = 44,     // [distinct, scope, name, variable,...]
   METADATA_GENERIC_SUBRANGE = 45 // [distinct, count, lo, up, stride]
```

Comment:
I am not sure about re-numbering as I am afraid it may break backward compatibility. We may leave the gap to be used in future. Please let me know your opinion ?

Reply:
Ah, I didn't realize that 42 and 43 had already been reserved in trunk! My impression was that these enums were changed quite freely in older releases of Classic Flang, but I do appreciate the exercise of caution here. LGTM.